docs/src/budyko_sellers_halfar.md

```diff
@@ -338,50 +346,58 @@ We can save the solution file to examine later.
 ## Visualize
 
 Quickly examine the final conditions for temperature:
+
 ``` @example DEC
-lines(map(x -> x[1], point(s′)), soln(tₑ).Tₛ)
+lines(map(x -> x[1], point(s)), soln(tₑ).Tₛ)
 ```
 
 Quickly examine the final conditions for ice height:
+
 ``` @example DEC
-lines(map(x -> x[1], point(s′)), soln(tₑ).halfar_dynamics_h)
+lines(map(x -> x[1], point(s)), soln(tₑ).halfar_dynamics_h)
 ```
 
-Create animated GIFs of the temperature and ice height dynamics:
-``` @example DEC
+<!-- Create animated GIFs of the temperature and ice height dynamics: -->
+``` @setup DEC
 begin
 # Initial frame
 frames = 100
-fig = Figure(; size = (800, 800))
+fig = Figure()
 ax1 = CairoMakie.Axis(fig[1,1])
-xlims!(ax1, extrema(map(x -> x[1], point(s′))))
+xlims!(ax1, extrema(map(x -> x[1], point(s))))
 ylims!(ax1, extrema(soln(tₑ).Tₛ))
 Label(fig[1,1,Top()], "Surface temperature, Tₛ, [C°]")
-Label(fig[2,1,Top()], "Line plot of temperature from North to South pole, every $(tₑ/frames) time units")
```

Comment:
Why is this removed

Reply:
It was making the plots show up with a lot of white space at the bottom. You can check it out here: https://algebraicjulia.github.io/Decapodes.jl/dev/budyko_sellers_halfar/

Reply:
What happens when you change the 800,800 to a rectangle

Reply:
I tried and it's the same. The problem is that another entire portion of the figure is made but is only filled with some text.

Reply:
I think the problem is `Label(fig[2,1,Top()]` should be `Label(fig[1,1,Bottom()]`

Reply:
I tried this but now it just overlaps with the bottom axis:
![image](https://github.com/AlgebraicJulia/Decapodes.jl/assets/78235421/f3a970dc-97a9-4cad-99ba-ff051e4a355d)

Reply:
I'm going to try messing around with the size of the figure but if that doesn't work I'll leave it as is now.

Reply:
Ok got it, we just had to use `xlabel` on the axis.